ansible/configs/ocp4-workshop/lifecycle.yml

```diff
@@ -77,7 +78,7 @@
         filters:
           "tag:guid": "{{ guid }}"
           "tag:env_type": "{{ env_type }}"
-
+          instance-state-name: stopped
```

Comment:
same here

Reply:
same comment....if we try to start things that are terminated we run into problems

Reply:
i guess this happens only when developing and reusing the same guid